src/client/terminals/envCollectionActivation/shellIntegrationService.ts

```diff
@@ -86,6 +85,12 @@ export class ShellIntegrationService implements IShellIntegrationService {
             this.isDataWriteEventWorking = false;
             traceError('Unable to check if shell integration is active', ex);
         }
+        const isEnabled = this.workspaceService
+            .getConfiguration('terminal')
+            .get<boolean>('integrated.shellIntegration.enabled')!;
```

Comment:
`!` is generally bad, you could do something like this to be a little safer:

```suggestion
        const isEnabled = !!this.workspaceService
            .getConfiguration('terminal')
            .get('integrated.shellIntegration.enabled');
```